mypyc/build.py

```diff
@@ -534,6 +534,8 @@ def mypycify(
             "-Wno-unused-command-line-argument",
             "-Wno-unknown-warning-option",
             "-Wno-unused-but-set-variable",
+            "-Wno-cpp",
```

Comment:
We should probably avoid adding to this list. Can you explain why these warnings need to be ignored?

Reply:
See https://github.com/mypyc/mypyc/issues/956

Reply:
What about moving this change to a separate PR? It looks like fixing the issue could be helpful on master, or it may make sense to cherry-pick this to a point release. We can also discuss if there is another way around the error.

Reply:
Ok, I will open a separate PR for this!